static/app/components/replays/replayCurrentUrl.tsx

```diff
@@ -27,6 +29,20 @@ function ReplayCurrentUrl() {
     );
   }
 
+  if (url.includes('[Filtered]')) {
+    return (
+      <Tooltip
+        title={t(
+          'This function is disabled since the URL contains filtered content, which is not a valid path.'
```

Comment:
Open to suggestions on improving this wording/explanation!

Reply:
"Copy paste is disabled..."

Reply:
updated